Sources/Purchasing/StoreKit2/StoreKit2TransactionListener.swift

```diff
@@ -37,10 +37,11 @@ protocol StoreKit2TransactionListenerType: Sendable {
 
     func set(delegate: StoreKit2TransactionListenerDelegate) async
 
-    /// - Returns: `nil` `CustomerInfo` if purchases were not synced
-    /// - Throws: Error if purchase was not completed successfully
+    /// - Throws: ``ErrorCode`` if purchase was not completed successfully
```

Comment:
Technically these throw `PurchasesError` (for the most part, since it's impossible to guarantee that there aren't any other types).